core/src/mindustry/world/blocks/production/GenericSmelter.java

```diff
@@ -41,7 +49,7 @@ public void draw(Tile tile){
             Draw.tint(flameColor);
             Fill.circle(tile.drawx(), tile.drawy(), 3f + Mathf.absin(Time.time(), 5f, 2f) + cr);
             Draw.color(1f, 1f, 1f, entity.warmup);
```

Comment:
this line is now duplicate, you might want to move it above the first if, and call reset when both are done 👀